app/views/play/CampaignView.coco.coffee

```diff
@@ -23,7 +23,7 @@ PollModal = require 'views/play/modal/PollModal'
 AnnouncementModal = require 'views/play/modal/AnnouncementModal'
 LiveClassroomModal = require 'views/play/modal/LiveClassroomModal'
 Codequest2020Modal = require 'views/play/modal/Codequest2020Modal'
-MineModal = require 'views/core/MineModal' # Minecraft modal
+MineModal = require 'views/core/MineModal' # Roblox modal
```

Comment:
Ideally `MineModal` and `maybeShowMinecraftModal` and `userQualifiesForMinecraftModal` would have `Mine` and `MineCraft` replaced with `Roblox